python/cudaq/kernel/ast_bridge.py

```diff
@@ -2896,16 +2906,21 @@ def visit_Name(self, node):
 
         if node.id in self.capturedVars:
             # Only support a small subset of types here
+            complexType = type(1j)
```

Comment:
I think you can just use `complex` instead of getting the type of `1j`

Reply:
We should also probably check for `np.complex64`

Reply:
> I think you can just use complex instead of getting the type of 1j

We use complex dialect and it seems to override the complex type. Maybe worth decoupling in a separate PR...

> We should also probably check for np.complex64

Done.